reference/enterprise-api-ref/inventory.markdown

```diff
@@ -0,0 +1,255 @@
+| nl           | NOT LIKE  |
+| il           | ILIKE     |
+| nil          | NOT ILIKE |
+
+```
+curl -k --user admin:admin -X POST https://hub.cfengine.com/api/inventory  -H 'content-type: application/json'   -d '{  
+                                                                                                                                  "sort":"Host name",
+                                                                                                                                  "filter":{  
+                                                                                                                                     "Host name":{  
+                                                                                                                                        "l":"%value1%",
+                                                                                                                                        "nil":"%value2%"
+                                                                                                                                     }
+                                                                                                                                  },
+                                                                                                                                  "select":[  
+                                                                                                                                     "Host name",
+                                                                                                                                     "OS type",
+                                                                                                                                     "IPv4 addresses",
+                                                                                                                                     "CFEngine version",
+                                                                                                                                     "Ports listening"
+                                                                                                                                  ],
+                                                                                                                                  "hostContextExclude":[  
+                                                                                                                                     "policy_server"
+                                                                                                                                  ]
+                                                                                                                               }
+'
+```
+
+**Example Request Body:**
+
+
+```
+{  
+   "sort":"Host name",
+   "filter":{  
+      "Host name":{  
+         "l":"%value1%",
```

Comment:
It took me some fiddling to understand what the % here meant.

Was it required on both ends as shown, is it automatically inserted around value. Maybe thats just my lack of experience around API docs.

I did get it working:
If % isnt used then it must be an exact match for hostname.
it can be used for wildcard on front or back or both.

Reply:
@nickanderson  I have intent to change operators, it will probably 
'<', '>','!=','=','<=', '>=', 'match', 'not_match', 'contains', 'not_contains'.
it gives possibility to avoid using %, back-end will handle this cases by himself.

Reply:
cool